src/Microsoft.Framework.SecretManager/Program.cs

```diff
@@ -187,6 +190,19 @@ public int Main(string[] args)
             }
         }
 
+        private static string GetInformationalVersion()
+        {
+            var assembly = typeof(Program).GetTypeInfo().Assembly;
+            var attributes = assembly.GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute)) as AssemblyInformationalVersionAttribute[];
+
+            var versionAttribute = string.Empty;
+            versionAttribute = attributes.Length == 0 ? 
```

Comment:
You can do all the assignment as part of the variable declaration. No need to split it across two statements.